stages/org.osbuild.xorrisofs.meta.json

```diff
@@ -54,6 +54,10 @@
         "efi": {
           "type": "string"
         },
+        "grub2": {
+          "type": "string",
```

Comment:
Yeah, I was waffling between making this a bool or passing in the path and decided that the path was more flexible. Going to call it grub2mbr so that it is more similar to isohybridmbr